lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.js

```diff
@@ -168,6 +180,11 @@ function handleLogs() {
     const region = this.options.region;
     const logGroupName = `/aws/api-gateway/${service}-${stage}`;
 
+    let logFormat = defaultApiGatewayLogFormat;
+    if (provider.logs.restApi.logFormat) {
```

Comment:
Wouldn't want it to crash if `provider.logs.restApi` isn't an object: 
```suggestion
    if (provider.logs.restApi && provider.logs.restApi.logFormat) {
```

Reply:
D'oh, thank you

Reply:
@alexdebrie  Here `logs = provider.logs.restApi` and `logs` is guaranteed to be truthy, so there's no way it can fail.

It'll be just cleaner to rely on `if (logs.logFormat)` (or `logs.format`)